src/modules/video_player/index.js

```diff
@@ -80,6 +81,18 @@ class VideoPlayerModule {
         watcher.on('load.player', () => {
             this.clickToPause();
             watchPlayerRecommendationVodsAutoplay();
+
+            const currentPlayer = twitch.getCurrentPlayer();
+            this.isMutedByUser =
```

Comment:
we probably want to store the value of muted on blur/invisible .. then restore to that value on focus/visible